types/src/client.rs

```diff
@@ -6,7 +6,8 @@ use serde::de::DeserializeOwned;
 use serde_json::Value as JsonValue;
 
 /// Active subscription on a Client.
-pub struct Subscription<Notif> {
+#[derive(Debug)]
+pub struct Subscription<Notif: std::fmt::Debug> {
```

Comment:
I've mixed feelings about this, I like having strict lints, but this looks really noisy to me and I'm not sure there is much value to it.

Also it doesn't look like the bound is really necessary since it's only applied to `PhantomData`?

Reply:
> (Sidenote: why is PhantomData pub?)

it because we have no constructor for it, I have fixed it in #356

Reply:
we could do a manual debug implementation for this instead.

Reply:
Makes sense, replaced it with the manual implementation.